tests/components/homematicip_cloud/test_init.py

```diff
@@ -114,14 +97,17 @@
         instance.home.modelType = "mock-type"
         instance.home.name = "mock-name"
         instance.home.currentAPVersion = "mock-ap-version"
+        instance.async_reset.return_value = mock_coro(True)
```

Comment:
Set `async_reset` to a `CoroutineMock` from asynctest instead of using `mock_coro`.